tutorials/intro/plot_introduction.py

```diff
@@ -1,404 +1,374 @@
+# use :meth:`~mne.io.Raw.load_data` first. We'll also make a copy of the
+# :class:`~mne.io.Raw` object so we can compare the signal before and after
+# artifact removal side-by-side:
+
+orig_raw = raw.copy()
+raw.load_data()
+ica.apply(raw)
+
+# show some frontal channels to clearly illustrate the artifact removal
+chs = ['MEG 0111', 'MEG 0121', 'MEG 0131', 'MEG 0211', 'MEG 0221', 'MEG 0231',
+       'MEG 0311', 'MEG 0321', 'MEG 0331', 'MEG 1511', 'MEG 1521', 'MEG 1531',
+       'EEG 001', 'EEG 002', 'EEG 003', 'EEG 004', 'EEG 005', 'EEG 006',
+       'EEG 007', 'EEG 008']
+chan_idxs = [raw.ch_names.index(ch) for ch in chs]
+orig_raw.plot(order=chan_idxs, start=12, duration=4)
+raw.plot(order=chan_idxs, start=12, duration=4)
+
+###############################################################################
+# Detecting experimental events
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 #
-# First extract events:
+# The sample dataset includes several "STIM" channels that recorded electrical
+# signals sent from the stimulus delivery computer (as brief DC shifts /
+# squarewave pulses). These pulses (often called "triggers") are used in this
+# dataset to mark experimental events: stimulus onset, stimulus type, and
+# participant response (button press). The individual STIM channels are
+# combined onto a single channel, in such a way that voltage
+# levels on that channel can be unambiguously decoded as a particular event
+# type. On older Neuromag systems (such as that used to record the sample data)
+# this summation channel was called ``STI 014``, so we can pass that channel
+# name to the :func:`mne.find_events` function to recover the timing and
+# identity of the stimulus events.
 
 events = mne.find_events(raw, stim_channel='STI 014')
-print(events[:5])
-
-##############################################################################
-# Note that, by default, we use stim_channel='STI 014'. If you have a different
-# system (e.g., a newer system that uses channel 'STI101' by default), you can
-# use the following to set the default stim channel to use for finding events:
-
-mne.set_config('MNE_STIM_CHANNEL', 'STI101', set_env=True)
-
-##############################################################################
-# Events are stored as a 2D numpy array where the first column is the time
-# instant and the last one is the event number. It is therefore easy to
-# manipulate.
+print(events[:5])  # show the first 5
+
+###############################################################################
+# The resulting events array is an ordinary 3-column :class:`NumPy array
+# <numpy.ndarray>`, with sample number in the first column and integer event ID
+# in the last column; the middle column is usually ignored. Rather than keeping
+# track of integer event IDs, we can provide an *event dictionary* that maps
+# the integer IDs to experimental conditions or events. In this dataset, the
+# mapping looks like this:
 #
```

Comment:
I would also cross reference this paragraph.

Reply:
@massich do you mean add a label like `.. _event-dict-table` as a cross-reference target?